src/main/java/jackson/utils/Ui.java

```diff
@@ -0,0 +1,213 @@
+        String output = String.format("Alamak, you got %d items on the list only leh...\n", size);
+
+        // different responses based on taskList size
+        if (size == 0) {
+            output += "You've got no items in the list! Add some stuff first!";
+        } else if (size == 1) {
+            output += "Enter 1 to mark/unmark/delete the task in the list!";
+        } else {
+            output += String.format("Enter a number between 1 and %d when marking/unmarking/deleting tasks!\n", size);
+        }
+        return output + "\n";
+    }
+
+    /**
+     * Returns generic error message when other Exceptions occur.
+     * @param e {@code Exception} encountered during runtime.
+     * @return String response.
+     */
+    public String printUnknownError(Exception e) {
+        return String.format("Oops! Something went wrong!\n%s\n", e.toString());
+    }
+
+    /**
+     * Returns chatbot response when adding task to list.
+     * @param task {@code Task} object to be added to taskList.
+     * @param taskList {@code TaskList} object task is to be added to.
+     * @return String response.
+     */
+    public String printAddList(Task task, TaskList taskList) {
+        String output = "";
+        output += "Ya la, adding this task to your list!\n";
+        output += String.format("\t%s\n", task);
+        output += String.format("You now got %s tasks in your list leh\n", taskList.getSize());
+        return output;
+    }
+
+    /**
+     * Returns chatbot response when deleting task from list.
+     * @param task {@code Task} object to be deleted from taskList.
+     * @param taskList {@code TaskList} object task is to be deleted from.
+     * @return String response.
+     */
+    public String printDeleteList(Task task, TaskList taskList) {
+        String output = "";
+        output += "Deleting now hor!\n";
+        output += String.format("\t%s\n", task);
+        output += String.format("You now got %s tasks in your list leh\n", taskList.getSize());
+        return output;
+    }
+
+    /**
+     * Returns chatbot response when finding tasks containing keyword.
+     * @param tasks {@code ArrayList} of tasks that contain keyword.
+     * @param keyword String keyword.
+     * @return String response.
+     */
+    public String printFindList(ArrayList<Task> tasks, String keyword) {
+        String output = "";
+        output += "Ok ok, find for you already...\n";
+        if (tasks.isEmpty()) {
+            output += String.format("Don't have tasks that match '%s' leh....\n", keyword);
+        } else {
+            for (int i = 0; i < tasks.size(); i++) {
+                output += String.format("%d. %s\n", i + 1, tasks.get(i));
+            }
+        }
+        return output;
+    }
+
+    /**
+     * Returns chatbot response when marking task in list.
+     * @param task {@code Task} object to be marked.
+     * @return String response.
+     */
+    public String printMark(Task task) {
```

Comment:
Maybe we could rename this to show that we are printing Marked Task instead of printing Mark (could be misinterpreted)?